src/Modules/Dashboard/DashBoardDataComponent.js

```diff
@@ -1,35 +1,40 @@
-import React from 'react';
-import { withRouter } from 'react-router-dom';
+import * as React from 'react';
```

Comment:
What is the difference with these changes? Do we want to import each resource explicitly instead of by '*'?